.github/workflows/ci-llama-large-tests.yaml

```diff
@@ -55,18 +55,12 @@ jobs:
           python -m pip install --no-compile --upgrade pip
 
           # Note: We install in three steps in order to satisfy requirements
-          # from non default locations first. Installing the PyTorch CPU
-          # wheels saves multiple minutes and a lot of bandwidth on runner setup.
+          # from non default locations first.
           pip install --no-compile -r pytorch-cpu-requirements.txt
-
-          # Install nightly IREE packages.
-          # We could also pin to a known working or stable version.
-          pip install -f https://iree.dev/pip-release-links.html --pre --upgrade \
-            iree-base-compiler \
-            iree-base-runtime \
-            iree-turbine
-
-          pip install --no-compile -r requirements.txt -r sharktank/requirements-tests.txt -e sharktank/
```

Comment:
With dropping `requirements.txt` (which I am highly supportive for), this also drops installing [requirements-dev.txt](https://github.com/nod-ai/shark-ai/blob/main/requirements-dev.txt) and (probably more important) [sharktank/requirements.txt](https://github.com/nod-ai/shark-ai/blob/main/sharktank/requirements.txt). The latter might be pulled in via `-e sharktank/` as the requirements file is specified in sharktank's `pyproject.toml`.

Reply:
Yeah I think `sharktank/requirements-tests.txt` and `-e sharktank/` should be sufficient, given that sharktank includes its deps/requirements.